Makefile

```diff
@@ -242,9 +242,8 @@ localnet_regenesis: check_yq acc_initialize_pubkeys_warn_message ## Regenerate t
 	@echo "Initializing chain..."
 	@set -e ;\
 	ignite chain init --skip-proto ;\
-	mkdir -p $(POKTROLLD_HOME)/config/ ;\
 	cp -r ${HOME}/.poktroll/keyring-test $(POKTROLLD_HOME) ;\
-	cp -r ${HOME}/.poktroll/config/ $(POKTROLLD_HOME)/config/ ;\
+	cp -r ${HOME}/.poktroll/config $(POKTROLLD_HOME)/ ;\
```

Comment:
I found a bunch of other references to `/config/`. 

1. Are you confident that this works?
2. Does any of the other code need to be modified?

![Screenshot 2024-03-12 at 3 28 18 PM](https://github.com/pokt-network/poktroll/assets/1892194/6f7ef02d-dc35-4b23-9a93-e2b8991bfada)
![Screenshot 2024-03-12 at 3 28 08 PM](https://github.com/pokt-network/poktroll/assets/1892194/a7f2b4be-717f-4978-898c-22eb18ddad2b)

Reply:
On the first screenshot, the copy is commented but is fine too. Since each copy command copies a single file.

The second screenshot is using ` $(POKTROLLD_HOME)/config/` and not ` $(POKTROLLD_HOME)/config/config` which is fine too.

I'm confident we changed them only where needed.